index.html

```diff
@@ -528,6 +538,54 @@ <h3>image</h2>
         </table>
 
       </section>
+      <section class="normative">
+        <h3>imageHash</h2>
+
+        <p>
+          A digest value which uniquely identifies the representative image of the individual
+        </p>
+
+        <table class="simple">
+          <tbody>
+            <tr>
+              <td>
+Term
+              </td>
+              <td>
+imageHash
+              </td>
+            </tr>
+            <tr>
+              <td>
+URL
+              </td>
+              <td>
+https://schema.org#imageHash
```

Comment:
This doesn't appear to be defined at schema.org.

Reply:
Hi David, you are right, imageHash is not defined at schema.org. Would it be correct to  change the url pointing to https://w3id.org/citizenship#imageHash ? 
I've noticed there are other urls that are not defined at schema.org like https://schema.org#birthCountry . Should it be replaced with https://schema.org/birthPlace

Reply:
> Would it be correct to change the url pointing to https://w3id.org/citizenship#imageHash ?

Yes, that would be the right thing to do, although there is an issue w/ imageHash. Will explain in a separate comment.